SteamBotUnitTest/SteamTrade/GenericInventoryTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using SteamKit2;
+using SteamTrade;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SteamBotUnitTest.SteamTrade
+{
+    [TestFixture]
+    class GenericInventoryTest
+    {
+        [Test]
+        public void LoadTest()
+        {
+            var inventory = new GenericInventory(new SteamWeb());
+            //76561198104350201 is my friend's Steam account with lots of Dota2 items which will test the start/more way of loading. It's inventory will remain public and contain more than 2000 items long term.
```

Comment:
I can guarantee that this steam account's inventory will remain public and valid (has more than 2000 items) in the foreseeable future (I'm talking about a few years here).

I can also remove the test if you want. But i think this is better than no test at all. After all, steam may change their internal API and this will be a good way to discover it.